### ISSUE TITLE
Unexpected character: '#'

### ISSUE THREAD

**Issue by ds1371dani** [developit/rollup-plugin-preserve-shebang#11]
hi, when i try using this plugin i get following error:
```
[!] Error: Unexpected character '#' (Note that you need plugins to import files that are not JavaScript)
```
this is my rollup config:
```
import commonjs from "@rollup/plugin-commonjs";
import json from "@rollup/plugin-json";
import { terser } from "rollup-plugin-terser";
import shebang from "rollup-plugin-preserve-shebang";
import typescriptPlugin from "rollup-plugin-typescript2";
import typescript from "typescript";
import pkg from "./package.json";

const index = {
  external: [...Object.keys(pkg.peerDependencies || {}), "child_process", "path", "fs"],
  input: "src/index.ts",
  output: [
    {
      file: pkg.main,
      format: "cjs",
      sourcemap: true
    }
  ],
  plugins: [
    shebang(),
    commonjs(),
    typescriptPlugin({
      typescript
    }),
    json(),
    terser()
  ]
};

export default [index];

```

**Comment by revelt** [comment]
Was it a plugin error or rollup error? For example, I'm bundling a CLI app and `shebang()` passes through the shebang all right, however `@rollup/plugin-strip` chokes upon shebang..

It's hard to advise without seeing the original code but in your case I'd back up everything and assemble the minimal case of just shebang and something trivial like `console.log`. Then add plugins piece by piece, starting with TS. You know, _divide and conquer_.

**Comment by ds1371dani** [comment]
> Was it a plugin error or rollup error? For example, I'm bundling a CLI app and `shebang()` passes through the shebang all right, however `@rollup/plugin-strip` chokes upon shebang..
> 
> It's hard to advise without seeing the original code but in your case I'd back up everything and assemble the minimal case of just shebang and something trivial like `console.log`. Then add plugins piece by piece, starting with TS. You know, _divide and conquer_.

the error is from this plugin because when i tested it with a single file containing shebang and some logging and only this plugin for rollup, i had the same error.
maybe there is a version problem because there is a warning too when i run rollup
`(!) The "transformBundle" hook used by plugin preserve-shebang is deprecated. The "renderChunk" hook should be used instead.`
anyway i switched to `rollup-plugin-hashbang`. it didn't have any problem.

**Comment by developit** [comment]
This should be fixed in [1.0.0](https://github.com/developit/rollup-plugin-preserve-shebang/releases/tag/1.0.0).
